services/cargohold/src/CargoHold/Options.hs

```diff
@@ -51,11 +52,24 @@ data AWSOpts = AWSOpts
     _awsS3DownloadEndpoint :: !(Maybe AWSEndpoint),
     -- | S3 bucket name
     _awsS3Bucket :: !Text,
+    -- | Enable this option for compatibility with specific S3 backends.
+    _awsS3Compatibility :: !(Maybe (Last S3Compatibility)),
```

Comment:
Just wondering: Why wrap this in `Last` ?

Reply:
Ah, yeah, I saw it used in other options and was wondering about it as well. Forgot to remove it, but it's done now and apparently didn't have any deeper reason.